src/openforms/js/components/form/content.js

```diff
@@ -26,19 +26,80 @@ const CONTENT_PRESENTATION = {
   components: [SHOW_IN_EMAIL, SHOW_IN_PDF],
 };
 
+const CONTENT_EDITOR = {
+  weight: 0,
+  type: 'textarea',
+  editor: 'ckeditor',
+  label: 'Content',
+  hideLabel: true,
+  input: true,
+  key: 'html',
+  as: 'html',
+  rows: 3,
+  tooltip: 'The HTML template for the result data items.',
+};
+
 const CONTENT_EDIT_TABS = {
   components: [
     {
-      weight: 0,
-      type: 'textarea',
-      editor: 'ckeditor',
-      label: 'Content',
-      hideLabel: true,
-      input: true,
-      key: 'html',
-      as: 'html',
-      rows: 3,
-      tooltip: 'The HTML template for the result data items.',
+      type: 'columns',
+      key: 'content',
+      columns: [
+        {
+          components: [
+            {
+              type: 'tabs',
+              components: [{label: 'Content', components: [CONTENT_EDITOR]}],
```

Comment:
Single tab. The easiest way to make it align and look similar side by side to the translations.